KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,47 @@
+# Kubeflow Outreach Committee (KOC)
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help onboard new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
+
+### Committee Structure
+
+- **Appointment of Members**: Members of the Kubeflow Outreach Committee are appointed at the discretion of the Kubeflow Steering Committee (KSC), with the removal of any members for violating the CNCF Code of Conduct also at the KSC's discretion.
+- **Role of KOC Co-Leads**: The Outreach Committee will be co-led by established Kubeflow community members, with the co-leads being responsible for guiding the committee’s work. New members of the KOC may be newcomers to the community, and their primary role will be focused on helping drive growth for Kubeflow.
+- **Committee Expansion**: KOC may appoint new members as needed, requiring only the approval of one KSC member for the appointment.
+
+### Committee Meetings
+
+The KOC will meet regularly (at least monthly, or as needed), and these meetings will be open to the public. Specific sensitive matters, such as code of conduct violations or confidential discussions, may be handled separately.
+
+Meeting notes will be publicly available to the community, except when privacy is required for specific discussions.
+
+### Decision Process
+
+The KOC aims to reach consensus on all decisions. If a decision requires a vote, the decision will pass with a simple majority of the members present.
+
+### Special Decision Process
+
+In some cases, such as changes to the KOC’s charter or significant shifts in community direction, a special decision process may apply. A proposal can pass with 70% approval from the committee members present.
+
+### Committee Members
```

Comment:
There should probably be some limitations of company representation, [similar to the KSC](https://github.com/kubeflow/community/tree/master/proposals/645-kubeflow-steering-committee-election#limitations-on-company-representation).

Reply:
Yeah that's fine.